enhancements/machine-api/aws_efa_networking.md

```diff
@@ -0,0 +1,253 @@
+---
+title: aws-efa-networking
+authors:
+  - "@JoelSpeed"
+reviewers:
+  - "@elmiko"
+  - "@deads2k"
+  - TBD
+approvers:
+  - TBD
```

Comment:
Lets define who's an approver to make things more clear.

Reply:
You can add @deads2k and me here.

Reply:
Add me too please.